src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -112,6 +116,7 @@ class DraftAdjudicationService(
       .toDto(offenceCodeLookupService)
   }
 
+  @Transactional
```

Comment:
you might be able to move this to class level annotation if you want everything to be transactional

Reply:
yeah. Was thinking the same.  I inherited the function level annotations, so have been sticking with it, though it would have reduced a lot of the hibernate and other type errors.

Reply:
i think i will raise a TD ticket as there are a number of things that should really be addressed and this is one of them.